language/move-prover/src/prelude.bpl

```diff
@@ -98,6 +98,8 @@ function {:constructor} Address(a: Address): Value;
 function {:constructor} ByteArray(b: ByteArray): Value;
 function {:constructor} Str(a: String): Value;
 function {:constructor} Vector(v: ValueArray): Value; // used to both represent move Struct and Vector
+// DD
+function {:constructor} $range(lb: Value, ub: Value): Value;
```

Comment:
Remove DD? Also I think we should $Range for consistency (I know this file is terribly inconsistent regards naming, but constructors at least use capital first letter everywhere,